src/vcpkg/base/system.process.cpp

```diff
@@ -613,12 +615,25 @@ namespace vcpkg
             "CUDA_PATH",
             "CUDA_PATH_V9_0",
             "CUDA_PATH_V9_1",
+            "CUDA_PATH_V9_2",
             "CUDA_PATH_V10_0",
             "CUDA_PATH_V10_1",
             "CUDA_PATH_V10_2",
             "CUDA_PATH_V11_0",
             "CUDA_PATH_V11_1",
             "CUDA_PATH_V11_2",
+            "CUDA_PATH_V11_3",
+            "CUDA_PATH_V11_4",
+            "CUDA_PATH_V11_5",
+            "CUDA_PATH_V11_6",
+            "CUDA_PATH_V11_7",
+            "CUDA_PATH_V11_8",
+            "CUDA_PATH_V12_0",
+            "CUDA_PATH_V12_1",
+            "CUDA_PATH_V12_2",
+            "CUDA_PATH_V12_3",
+            "CUDA_PATH_V12_3",
+            "CUDA_PATH_V12_6",
```

Comment:
Maybe it makes sense to allow all vars that's starts with `CUDA_PATH`

Reply:
if you know how to do it, yes your suggestion is perfect and future proof!

Reply:
I will create a PR :)

Reply:
do you think it could be ok to merge this asap while you work on your improved version? especially for the proxy support env variable...

Reply:
Have created #1476

Reply:
maybe it makes sense to only allow `CUDA_PATH` since that is the only true value.